index.html

```diff
@@ -204,9 +204,9 @@ <h3>Expected Behavior</h3>
 
 <li>Think of others’ needs from their point of view. Use preferred names, titles (including pronouns) and the appropriate tone of voice. Therefore, be formal and conservative in what you do and liberal in what you accept from others, at least until a truly friendly atmosphere and relationships are established.</li>
 
-<li>Be sensitive to language differences. English is the default language of the W3C. However, only some of us are native English speakers. Many participants speak English as a second (third, …). People who communicate in non-native language often struggle to understand fast and/or quiet speech, and tend to speak louder than they usually would when communicating in their native tongue. If someone struggles to express his thoughts, help ensure their ideas are adequately expressed, heard, and granted thorough consideration.</li>
+<li>Be sensitive to language differences. English is the default language of the W3C. However, only some of us are native English speakers. Many participants speak English as a second (or third). People who communicate in non-native language often struggle to understand fast and/or quiet speech, and tend to speak louder than they usually would when communicating in their native tongue. If someone struggles to express his thoughts, help ensure their ideas are adequately expressed, heard, and granted thorough consideration.</li>
```

Comment:
The sentence "Many participants speak English as a second (or third)." still doesn't make sense - grammatically speaking, it lacks an object.

Reply:
I suspect that the part of the original text got accidentally deleted, because the rest of it is preserved in its entirety. The original part of the PWETF draft has this:
"English is a language of choice for many. However, only some of us are native English speakers. Many participants speak English as a second (third, …) language, and may not possess the same level of fluency or eloquence required to express complex concepts and ideas. People who communicate in non-native language often struggle to understand fast and/or quiet speech, and tend to speak louder than they usually would when communicating in their native tongue. If someone struggles to express his thoughts, help ensure their ideas are adequately expressed, heard, and granted thorough consideration."

Reply:
This was fixed in adf2b5c by @AdaRoseCannon .